_guides/plugin-creation.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: documentation
+title: Create a custom plugin
+description: Customize The Lounge using built-in plugin support, and share plugins with others
+---
+
+Plugins for The Lounge are npm packages hosted on [the npm registry](https://www.npmjs.com).
+The work similarly to themes. A theme is a plugin but not every plugin is a theme.
+
+In a directory named after your new plugin (for example, `thelounge-plugin-foo`), start by creating a new package with the following command:

```

Comment:
Maybe instead of this info, it might be better if we create a blank template for packages as a repository in the org? How does that sound to you? Then we can have a bit more info there around exactly what all this stuff is doing, but instead of all this explanation in this guide for setting up a dir, using yarn, etc, we can just say "clone [this repo]()" and go from there?

Reply:
Mmh, yeah, that would work too. I was just adapting the themes guide. Do we want to for the same for that too?